dispatches/models/fossil_case/properties/hitecsalt_properties.py

```diff
@@ -0,0 +1,424 @@
+                _log.info('{} Initialisation Step 1 Complete.'
+                          .format(blk.name))
+            else:
+                _log.warning('{} Initialisation Step 1 Failed.'
+                             .format(blk.name))
+
+        # ---------------------------------------------------------------------
+        if state_vars_fixed is False:
+            # release state vars fixed during initialization if control
+            # volume didn't fix the state vars
+            if hold_state is True:
+                return flags
+            else:
+                blk.release_state(flags)
+
+        if outlvl > 0:
+            if outlvl > 0:
+                _log.info('{} Initialisation Complete.'.format(blk.name))
+
+    def release_state(blk, flags, outlvl=0):
+        # Method to release states only if explicitly called
+
+        if flags is None:
+            return
+
+        # Unfix state variables
+        for k in blk.keys():
+            if flags['Fflag'][k] is False:
+                blk[k].flow_mass.unfix()
+            if flags['Pflag'][k] is False:
+                blk[k].pressure.unfix()
+            if flags['Tflag'][k] is False:
+                blk[k].temperature.unfix()
+
+        if outlvl > 0:
+            if outlvl > 0:
+                _log.info('{} State Released.'.format(blk.name))
+
+@declare_process_block_class("HitecsaltStateBlock", block_class=_StateBlock)
+class StateTestBlockData(StateBlockData):
+    """An example property package for Molten Salt properties."""
+
+    def build(self):
+        """Callable method for Block construction."""
+        super(StateTestBlockData, self).build()
+        self._make_params()
+        self._make_state_vars()
+        self._make_prop_vars()
+        self._make_constraints()
+
+    def _make_params(self):
+        """Make references to the necessary parameters contained."""
+        # Thermodynamic reference state
+        add_object_reference(self, "ref_temperature",
+                             self.config.parameters.ref_temperature)
+        # Density Coefficients
+        add_object_reference(self, "rho_param",
+                             self.config.parameters.rho_param)
+        # Specific Enthalpy Coefficients
+        add_object_reference(self, "cp_param",
+                             self.config.parameters.cp_param)
+        # Dynamic Viscosity Coefficients
+        add_object_reference(self, "mu_param",
+                             self.config.parameters.mu_param)
+        # Thermal Conductivity Coefficients
+        add_object_reference(self, "kappa_param",
+                             self.config.parameters.kappa_param)
+
+    def _make_state_vars(self):
+        """Declare the necessary state variable objects."""
+        self.flow_mass = Var(domain=Reals,
+                            initialize=100,
+                            units=pyunits.kg/pyunits.s,
+                            doc='Fluid mass flowrate [kg/s]')
```

Comment:
I generally suggest not including units in the doc strings now that we have explicit support for them.

Reply:
addressed